dynamic_programming/sum_of_subset.py

```diff
@@ -25,10 +26,40 @@ def isSumSubset(arr, arrLen, requiredSum):
 
     return subset[arrLen][requiredSum]
 
+'''
+
+
+# Take Linear Space to Solve the problem 
+
+def my_submission(arr, arrLen, requiredSum):
+
+    # a subset value says 1 if that subset sum can be formed else 0
+    #initially no subsets can be formed hence False/0
+    subset = ([[False for i in range(requiredSum + 1)] for i in range(2)])
+
+    #for each arr value, a sum of zero(0) can be formed by not taking any element hence True/1
+    for i in range(2):
+        subset[i][0] = True
+
+    #sum is not zero and set is empty then false
+    for i in range(1, requiredSum + 1):
+        subset[0][i] = False
+
+    for i in range(1, arrLen + 1):
+        for j in range(1, requiredSum + 1):
+            if arr[i-1]>j:
+                subset[i%2][j] = subset[(i+1)%2][j]
+            if arr[i-1]<=j:
+                subset[i][j] = (subset[(i+1)%2][j] or subset[(i+1)%2][j-arr[i-1]])
+
+ 
+    return subset[arrLen%2][requiredSum]
+
+
 arr = [2, 4, 6, 8]
 requiredSum =  5
 arrLen = len(arr)
 if isSumSubset(arr, arrLen, requiredSum):
```

Comment:
You also changed the method name above, but forgot to update it here! So it throws an error: `undefined name 'isSumSubset'`